internal/controller/authenticator/jwtcachefiller/jwtcachefiller.go

```diff
@@ -179,33 +182,29 @@ func (c *jwtCacheFillerController) Sync(ctx controllerlib.Context) error {
 
 	// If this authenticator already exists, then only recreate it if is different from the desired
```

Comment:
Could we clarify this comment as well? Suggested:

```golang
// Only update the cache if this authenticator is different than the desired authenticator, to prevent adding new authenticators every time Sync is called
```

Reply:
Updated in the second commit on this PR.

Reply:
got it